diff --git a/triplets/arm-uwp.cmake b/triplets/arm-uwp.cmake
index f16f96a4a..28c80c383 100644
--- a/triplets/arm-uwp.cmake
+++ b/triplets/arm-uwp.cmake
@@ -1,6 +1,5 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-
-set(VCPKG_CMAKE_SYSTEM_NAME WindowsStore)
-set(VCPKG_CMAKE_SYSTEM_VERSION 10.0)
+set(VCPKG_TARGET_ARCHITECTURE arm)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+set(VCPKG_CMAKE_SYSTEM_NAME WindowsStore)
+set(VCPKG_CMAKE_SYSTEM_VERSION 10.0)
\ No newline at end of file
diff --git a/triplets/arm64-windows.cmake b/triplets/arm64-windows.cmake
index ac911c476..f798113e1 100644
--- a/triplets/arm64-windows.cmake
+++ b/triplets/arm64-windows.cmake
@@ -1,5 +1,3 @@
 set(VCPKG_TARGET_ARCHITECTURE arm64)
 set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-
-
+set(VCPKG_LIBRARY_LINKAGE static)
diff --git a/triplets/x64-linux.cmake b/triplets/x64-linux.cmake
index 519618463..7c81607a8 100644
--- a/triplets/x64-linux.cmake
+++ b/triplets/x64-linux.cmake
@@ -1,5 +1,4 @@
-set(VCPKG_TARGET_ARCHITECTURE x64)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE static)
-
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+set(VCPKG_CMAKE_SYSTEM_NAME Linux)
\ No newline at end of file
diff --git a/triplets/x64-uwp.cmake b/triplets/x64-uwp.cmake
index 944acdd83..34693e34b 100644
--- a/triplets/x64-uwp.cmake
+++ b/triplets/x64-uwp.cmake
@@ -1,6 +1,5 @@
-set(VCPKG_TARGET_ARCHITECTURE x64)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-
-set(VCPKG_CMAKE_SYSTEM_NAME WindowsStore)
-set(VCPKG_CMAKE_SYSTEM_VERSION 10.0)
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+set(VCPKG_CMAKE_SYSTEM_NAME WindowsStore)
+set(VCPKG_CMAKE_SYSTEM_VERSION 10.0)
\ No newline at end of file
diff --git a/triplets/x64-windows.cmake b/triplets/x64-windows.cmake
index d0be7297f..d7acd46a4 100644
--- a/triplets/x64-windows.cmake
+++ b/triplets/x64-windows.cmake
@@ -1,4 +1,3 @@
-set(VCPKG_TARGET_ARCHITECTURE x64)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
diff --git a/triplets/x86-windows.cmake b/triplets/x86-windows.cmake
index 40ccbe152..22abeaf2d 100644
--- a/triplets/x86-windows.cmake
+++ b/triplets/x86-windows.cmake
@@ -1,3 +1,3 @@
-set(VCPKG_TARGET_ARCHITECTURE x86)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
+set(VCPKG_TARGET_ARCHITECTURE x86)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
