diff --git a/ports/nanogui/portfile.cmake b/ports/nanogui/portfile.cmake
index bc868e549..b3c4e68f4 100644
--- a/ports/nanogui/portfile.cmake
+++ b/ports/nanogui/portfile.cmake
@@ -4,6 +4,12 @@ if(VCPKG_TARGET_IS_UWP)
     message(FATAL_ERROR "nanogui doesn't support UWP.")
 endif()
 
+if (VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)
+    set(NANOGUI_BUILD_SHARED ON)
+else()
+    set(NANOGUI_BUILD_SHARED OFF)
+endif()
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO wjakob/nanogui
@@ -16,6 +22,9 @@ vcpkg_from_github(
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
+     OPTIONS
+        -DNANOGUI_BUILD_SHARED=${NANOGUI_BUILD_SHARED}
+        -DNANOGUI_BUILD_EXAMPLE=OFF
 )
 
 vcpkg_install_cmake()
